docs/framework/templates/index.md

```diff
@@ -88,9 +88,11 @@ CiviCRM introduces some extra features to Smarty:
 
 All the templates are under the folder `templates/CRM` in your CiviCRM
 installation. Finding which template is used on a given page can be
-difficult, but the easiest way to find out the answer is to view the
-source of the page from a web browser and search for `.tpl`. For
-example, for the Contact Summary page, use the web browser to open the
+difficult, but the easiest way to find out the answer is to enable debugging
+then view the source of the page from a web browser and search for `.tpl`. 
+To enable debugging to go to **Administer > System Settings > Debugging 
+and Error Handling** then select **Enable Debugging**.
```

Comment:
Perhaps this should be a link to https://docs.civicrm.org/dev/en/latest/tools/debugging/ ?